packages/calypso-e2e/src/lib/components/editor-gutenberg-component.ts

```diff
@@ -147,7 +146,7 @@ export class EditorGutenbergComponent {
 	async getSelectedBlockElementHandle( blockEditorSelector: string ): Promise< ElementHandle > {
 		// Note the :is() selector. This is to support both the block API v1 and V2.
 		const locator = this.editorCanvas.locator(
-			`:is(${ editorPane } ${ blockEditorSelector }.is-selected, ${ editorPane } ${ blockEditorSelector }.has-child-selected)`
+			`:is(${ blockEditorSelector }.is-selected, ${ blockEditorSelector }.has-child-selected)`
```

Comment:
These selectors used to have the `editorPane` to narrow the target, but it ended up causing problems here as the DOM structure of the editor canvas has been in flux (block-based theme, etc).